src/main/server.ts

```diff
@@ -289,26 +294,41 @@ const unwrapArgs = function (sender: WebContents, frameId: number, contextId: st
 
 const isRemoteModuleEnabledImpl = function (contents: WebContents) {
   const webPreferences = contents.getLastWebPreferences() || {}
-  return webPreferences.enableRemoteModule != null ? !!webPreferences.enableRemoteModule : false
+  return (webPreferences as any).enableRemoteModule != null ? !!(webPreferences as any).enableRemoteModule : false
```

Comment:
Do you have to use "any" here? You can't use the interface of it ?

Reply:
There are breaking changes in `electron >= 14`, which drops the `enableRemoteModule` prop

Reply:
seen, thanks! Having this issue in a project. So waiting for this PR to be in.